validator/src/cli.rs

```diff
@@ -240,27 +254,33 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .long("enable-extended-tx-metadata-storage")
                 .requires("enable_rpc_transaction_history")
                 .takes_value(false)
-                .help("Include CPI inner instructions, logs, and return data in \
-                       the historical transaction info stored"),
+                .help(
+                    "Include CPI inner instructions, logs, and return data in the historical \
+                     transaction info stored",
+                ),
         )
         .arg(
             Arg::with_name("rpc_max_multiple_accounts")
                 .long("rpc-max-multiple-accounts")
                 .value_name("MAX ACCOUNTS")
                 .takes_value(true)
                 .default_value(&default_args.rpc_max_multiple_accounts)
-                .help("Override the default maximum accounts accepted by \
-                       the getMultipleAccounts JSON RPC method")
+                .help(
+                    "Override the default maximum accounts accepted by the getMultipleAccounts \
+                     JSON RPC method",
+                ),
         )
         .arg(
             Arg::with_name("health_check_slot_distance")
                 .long("health-check-slot-distance")
                 .value_name("SLOT_DISTANCE")
                 .takes_value(true)
                 .default_value(&default_args.health_check_slot_distance)
-                .help("Report this validator healthy if its latest optimistically confirmed slot \
-                       that has been replayed is no further behind than this number of slots from \
-                       the cluster latest optimistically confirmed slot")
+                .help(
+                    "Report this validator as healthy if its latest, replayed optimistically \
```

Comment:
```suggestion
                    "Report this validator as healthy if its latest replayed optimistically \
```
Nit, but the comma hung me up a bit